android/lib/src/main/java/com/ern/api/impl/navigation/ElectrodeCoreActivity.java

```diff
@@ -0,0 +1,164 @@
+package com.ern.api.impl.navigation;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.KeyEvent;
+import android.view.MenuItem;
+import android.view.View;
+
+import androidx.annotation.CallSuper;
+import androidx.annotation.LayoutRes;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.annotation.StringRes;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+
+import com.ern.api.impl.core.ElectrodeReactFragmentActivityDelegate;
+import com.ernnavigationApi.ern.model.NavigationBar;
+import com.facebook.react.ReactRootView;
+
+import org.json.JSONObject;
+
+public abstract class ElectrodeCoreActivity extends AppCompatActivity implements ElectrodeReactFragmentActivityDelegate.DataProvider, MiniAppNavRequestListener {
```

Comment:
Since this class will serve as a base class, how about naming it `ElectrodeBaseActivity`? Or, what does "core" mean in this context?

Reply:
renamed to Base.